### PR TITLE
chore: update @types/node and @primer/octicons

### PR DESCRIPTION
**Pull request by codebytere**
#### Description of Change

Closes https://github.com/electron/electron/issues/19020.

Updates our `@types/node` version to match our Node.js version, as well as updating our octicons package for both major version and namespace (`octicons` => `@primer/octicons`).

cc @MarshallOfSound 

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: none

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,19 +7,20 @@
     "@electron/docs-parser": "^0.2.2",
     "@electron/typescript-definitions": "^8.3.1",
     "@octokit/rest": "^16.3.2",
+    "@primer/octicons": "^9.1.1",
```
Comment (MarshallOfSound) [inline package.json]: I feel as though this will break a thing / require a change in the default app?

Comment (codebytere) [reply package.json]: i considered this also, but couldn't find one and nothing seemed broken 🤔

Comment (codebytere) [reply package.json]: yep, pushed!

**Comment by MarshallOfSound** [comment]
Does this still work?

https://github.com/electron/electron/blob/master/default_app/index.ts#L3

I can't remember how I set that up 😆

**Comment by codebytere** [comment]
Screen Shot 2019-06-27 at 8 57 07 PM
Looks shipshape to me!

**Comment by codebytere** [comment]
@nornagon i actually don't think we need to bp this since `@types/node` doesn't exist in 5 :)

**Comment by pushkin-** [comment]
@codebytere It doesn't? My dependency tree for 5.0.6 shows @types/node@10.14.10

**Comment by codebytere** [comment]
i could be mistaken! let me check again :)

**Comment by pushkin-** [comment]
@codebytere Do you know if this will make it into 5.0.7?

**Comment by codebytere** [comment]
@pushkin- no: see [here](https://github.com/electron/electron/blob/5-0-x/package.json)

**Pull request merged**